afew/tests/test_mailmover.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: ISC
+# Copyright (c) Johannes Löthberg <johannes@kyriasis.com>
```

Comment:
Are you fine with dropping the copyright line, for the same reasons as in https://github.com/systemd/systemd/issues/9320:
> Information about who contributed to which file is already covered by `git log` and` git blame` much better. The "copyright by" lines that added manually are often out of date and just unnecessary boilerplate. In particular, all works are covered by copyright without any explicit notice since the 1971 Berne Convention.


I also plan to remove the `# -*- coding: utf-8 -*-` line, as it's a default in Python 3 anyways, so only the `SPDX-License-Identifier: ISC` needs to be preserved.

Reply:
`# -*- coding: utf-8 -*-` dropped in #247.

Reply:
Done.